source/module_hamilt_pw/hamilt_pwdft/VNL_in_pw.cpp

```diff
@@ -443,6 +443,7 @@ void pseudopot_cell_vnl::getvnl(Device* ctx, const int& ik, std::complex<FPTYPE>
     resmem_var_op()(ctx, vkb1, nhm * npw, "VNL::vkb1");
 
     ModuleBase::Vector3<double>* _gk = new ModuleBase::Vector3<double>[npw];
+    ModuleBase::Memory::record("pseudopot_cell_vnl::getvnl",sizeof(ModuleBase::Vector3<double>)*npw+sizeof(int)*GlobalC::ucell.ntype*3);
```

Comment:
Please only record `gk` here, the key word imply this memory covered all memory used in this function.